packages/wallets/src/components/WalletListCardDetails/WalletListCardDetails.tsx

```diff
@@ -8,12 +8,17 @@ import WalletListCardDropdown from '../WalletListCardDropdown/WalletListCardDrop
 import './WalletListCardDetails.scss';
 
 const WalletListCardDetails: React.FC = () => {
-    const { data: activeWallet, isLoading } = useActiveWalletAccount();
-    const isDemo = activeWallet?.is_virtual;
+    const { data: activeWallet } = useActiveWalletAccount();
+    const isDemo: boolean = useMemo(() => {
+        if (typeof activeWallet?.is_virtual === 'boolean') {
+            return activeWallet.is_virtual;
+        }
+        return isDemo || false;
+    }, [activeWallet?.is_virtual]);
```

Comment:
The point is to keep the previous state of wallet being real/demo until we receive new data from the API
Using `useMemo` here is a bit expensive given the goal, but i can't think of an other way
The best approach is prob to be able to pass `keepPreviousData=true` to any API hook and have the return data not changing until the new one is received, instead of data being `undefined` while fetching / loading

Reply:
Yup, thought about this approach too, explicit is better indeed

Reply:
Changes made